br/pkg/lightning/config/config.go

```diff
@@ -214,10 +214,7 @@ func (d *DBStore) adjust(
 		if d.Port <= 0 {
 			d.Port = int(settings.Port)
 		}
-		if len(d.PdAddr) == 0 {
-			pdAddrs := strings.Split(settings.Path, ",")
-			d.PdAddr = pdAddrs[0] // FIXME support multiple PDs once importer can.
-		}
+		d.PdAddr = settings.Path
```

Comment:
we still need to first check `if len(d.PdAddr) == 0`, then do `d.PdAddr = settings.Path`